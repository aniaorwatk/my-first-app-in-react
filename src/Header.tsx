import react from "react"

function Header(){

    return(
        <header>
            <h1></h1>
        </header>
    )
}