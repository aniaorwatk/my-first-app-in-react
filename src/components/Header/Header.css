.header--box{
    padding: 2rem;
    background-color: aqua;
}