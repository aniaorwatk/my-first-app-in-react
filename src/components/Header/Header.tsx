import react, {FC} from 'react';
import {Box, Typography} from '@mui/material';
import './Header.css'

const Header: FC = () => {
  const title: string = 'Meet ReqRes users!'
  const subtitle: string = 'Application created with free ReqRes API'

    return(
      <>
        <Box className="header--box" >
        <Typography variant="h4" gutterBottom component="div">
        {title}
        </Typography> 
        <Typography variant="h5" gutterBottom component="div">
        {subtitle}
        </Typography>
        </Box>
      </>
  )
}

export default Header
