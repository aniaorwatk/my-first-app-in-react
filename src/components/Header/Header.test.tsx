import React, { FC } from 'react';
import { fireEvent, render, screen } from '@testing-library/react';
import Header from './Header';

describe('Header', () => {
    it('Should render', () => {
        const { container } = render(<Header />);
        expect(container).toMatchSnapshot();
    })
})