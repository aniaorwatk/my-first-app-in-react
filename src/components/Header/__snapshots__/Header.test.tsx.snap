// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Should render 1`] = `
<div>
  <div
    class="header--box MuiBox-root css-0"
  >
    <div
      class="MuiTypography-root MuiTypography-h4 MuiTypography-gutterBottom css-1vw6mcs-MuiTypography-root"
    >
      Meet ReqRes users!
    </div>
    <div
      class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom css-h93ljk-MuiTypography-root"
    >
      Application created with free ReqRes API
    </div>
  </div>
</div>
`;
